db/versions/0037.yml

```diff
@@ -15,14 +15,16 @@ methods:
           queue_provisioner_entities_load.partition_key,
           queue_provisioner_entities_load.row_key,
           entity_buf_encode(
-            jsonb_build_object(
-              'PartitionKey', encode_string_key(provisioner_id),
-              'RowKey', 'provisioner',
-              'provisionerId', provisioner_id,
-              'expires', expires,
-              'lastDateActive', last_date_active,
-              'stability', stability),
-            'description', description::text) as value,
+            entity_buf_encode(
+              jsonb_build_object(
+                'PartitionKey', encode_string_key(provisioner_id),
+                'RowKey', 'provisioner',
+                'provisionerId', provisioner_id,
+                'expires', expires,
+                'lastDateActive', last_date_active,
+                'stability', stability),
+              'description', description::text),
+            'actions', actions::text) as value,
```

Comment:
It looks like the lack of `actions` in the first commit is causing the tests to fail on that commit (I went back in my local copy to run them).  Even with `db/` checked out from the full PR, the tests in that commit still fail, suggesting that the queue will fail during the time between upgrading to this DB version 37 and deploying the new version of the JS services.